regulations/templates/regulations/tree/preamble.html

```diff
@@ -8,19 +8,4 @@
     {% if node.interp %}data-interp-id="{{node.interp.markup_id}}"{% endif %}
 >
     {% include "regulations/node.html" %}
-
-    <div class="comment-wrapper" data-section="{{ node.markup_id }}">
-      <div class="toggle">(click to toggle comment box)</div>
-      <div class="comment">
-        <h3>Enter your comment below:</h3>
-        <form>
-          <div class="editor-container"></div>
-
-          <div class="queued"></div>
-          <input type="file">
-
-          <button type="submit">Add to your comment</button>
-        </form>
-      </div>
-    </div>
 </li>
```

Comment:
Good call--we may need to reintroduce this at some point, or not. Deleting for now.